src/main/js/SSEConnection.js

```diff
@@ -151,15 +151,16 @@ SSEConnection.prototype = {
                 var listenUrl = sseConnection.jenkinsUrl + '/sse-gateway/listen/'
                     + encodeURIComponent(tabClientId);
 
+                var options = undefined;
                 if (sseConnection.configuration.sendSessionId) {
```

Comment:
Ok, so in non-test code, this setting is false, so this is why this only matters for tests?

Reply:
Essentially yes. It's required only for code that cannot handle the regular cookies (= test libraries in this case)